packages/cli/src/commands/new.ts

```diff
@@ -156,9 +156,22 @@ export class New extends Command {
           const spinner = log.spinner(log.withBrand("Initializing SQLite database")).start()
 
           try {
-            // Required in order for DATABASE_URL to be available
-            require("dotenv-expand")(require("dotenv-flow").config({silent: true}))
-            await require("./db").Db.run(["migrate", "--name", "Initial Migration"])
+            // dotenv-cli used for DATABASE_URL to be available
+            spawn.sync(
+              "npx",
+              [
+                "dotenv",
+                "-e",
+                ".env.local",
+                "prisma",
+                "migrate",
+                "dev",
+                "--preview-feature",
+                "--name",
+                "Initial Migration",
+              ],
+              {stdio: "inherit"},
+            )
```

Comment:
This replacement code for the env loading doesn't have feature parity (dotenv is limited compared to dotenv-flow).

So keep the `require("dotenv-expand")(require("dotenv-flow").config({silent: true}))` line and then use spawn to just call prisma.
```
spawn.sync('npx', ["prisma", ...], {stdio: "inherit"})
```